### PR TITLE
Update manual balances and other stuff

### LINKED ISSUE #3370: Refactor manual balances identifiers
## Abstract

Currently we use the label of manually tracked balances as identifiers. This makes this field non mutable. 

## Motivation

It makes sense that, if we display the label as data for the balance a user, this field could be modified

## Specification

A new id random id should be generated and work as identifier, leaving the label field free to be modified.

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #3370 

## Checklist

- [x] Update manual balances in frontend prior changes in backend with identifier.
- [x] Delete snapshot
- [x] Fix Chart
- [x] Fix upgrade to premium button not working in docker.

### CODE REVIEW

File: frontend/app/tests/unit/components/settings/account-balances.spec.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ import '../../i18n';
 Vue.use(Vuetify);
 
 describe('AccountBalances.vue', () => {
-  let wrapper: Wrapper<AccountBalances>;
+  let wrapper: Wrapper<any>;
```
Comment (kelsos) [inline frontend/app/tests/unit/components/settings/account-balances.spec.ts:15]: At some point we need to figure out how to properly type the composition API components, but probably after migrating to vue 3

**Review by LefterisJP — COMMENTED** [review]
So tested this. Need to fix a backend bug for deletion that i noticed. Also we need to not have the virtual snapshot deletable.

Finally perhaps add a new changelog for deleting snapshots pointing to the manipulation of snapshots issue. To properly close that issue we will also need editing.

**Review by kelsos — APPROVED** [review]
codewise look good, after fixing @LefterisJP comments we can merge

**Comment by lukicenturi** [comment]
The virtual snapshot is prevented from being clicked now @LefterisJP @kelsos

**Pull request merged**